glusterfs/map.jinja

```diff
@@ -9,14 +9,21 @@
     },
 }, merge=salt['pillar.get']('glusterfs:server')) %}
 
+{% if salt['grains.get']('init') == "systemd" -%}
+        {% set mount_defaults = "defaults" %}
```

Comment:
Yes, that's it :-)
Only please fix indent here to be 4 spaces to match indentation of this file.